examples/exp_configs/rl/multiagent/multiagent_i210.py

```diff
@@ -140,7 +155,7 @@
     env=EnvParams(
         horizon=HORIZON,
         sims_per_step=1,
-        warmup_steps=0,
+        warmup_steps=800,
```

Comment:
I chose this number as the amount of time needed before congestion sets in. This seems like something the wave calibration team should figure out.